src/main/java/use_case/cat_management/change_cat_hunger/ChangeCatHungerInteractor.java

```diff
@@ -21,7 +21,8 @@ public ChangeCatHungerInteractor(CatDataAccessInterface catDataAccessObject,
     @Override
     public void execute(ChangeCatHungerInputData changeCatHungerInputData) {
         // get the cat, cat object should already exist dues to create cat use case
-        final Cat cat = changeCatHungerInputData.getCat();
+        final Cat cat = catDataAccessObject.getCatByNameAndOwner(changeCatHungerInputData.getCatName(),
```

Comment:
Fix return count of hungerDecreaseCalculator and magic numbers (Checkstyle errors);
Add constants to Constants class

Might also be good to make method naming more consistent with ChangeCatHungerInteractor e.g., calculateHungerDecrease

Reply:
```suggestion
        int hungerDecreaseCalculator(int studySessionLength) {
            int hungerDecrease = 0;
            if ... {
                hungerDecrease = Constants.HUNGER_FOR_LESS_EQUAL_20;
            }
            else if ... {

            }
            else if ... {

            }
            else {
                
            }
            return hungerDecrease;
        }
```